pallets/duster/src/tests.rs

```diff
@@ -141,13 +131,6 @@ fn dust_account_native_works() {
 				amount: 500,
 			}
 			.into(),
-			orml_currencies::Event::Transferred {
-				currency_id,
-				from: *ALICE,
-				to: *TREASURY,
-				amount: 500,
-			}
-			.into(),
```

Comment:
There is a comment in the runtime that orml_currencies was forked to keep these events, but they seem to be removed here?

Reply:
I removed orml events that followed events from the `balances` or `tokens` pallets. So we test that transfers are executed.

Reply:
Ah, so `pallet_currencies` keeps *some* but not all events?

Reply:
It keeps all events, but I removed events from tests if there was an event from `balances` or `tokens` corresponding to the same transfer. A transfer emits two events, one from `balances/tokens` and one from `currencies`